src/meldog_bridge/meldog_bridge/multi_moteus_controller.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+import rclpy
+from rclpy.node import Node
+import asyncio
+import moteus
+from meldog_interfaces.msg import MultiMoteusControl
+from meldog_interfaces.msg import MultiMoteusState
+from meldog_interfaces.msg import MoteusState
+from meldog_interfaces.msg import MoteusControl
```

Comment:
Możesz to zaimportować w jednej linijce podając klasy po przecinkach
`from cośtam import Wiadomosc1, Wiadomosc2, etc.`